.github/workflows/enforceTestCoverage.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - name: Generate Coverage Report
         if: env.solidity_changed == 'true'
         run: |
-          forge coverage --report lcov --force --evm-version 'shanghai' --ir-minimum
+          forge coverage --report lcov --force --evm-version 'cancun' --ir-minimum
```

Comment:
if we update foundry.toml to cancun then we dont need an explicit flag anymore here (but I am fine with leaving it)

Reply:
updated